src/test/java/AirportTest.java

```diff
@@ -35,98 +35,75 @@
+    public void testGetPassengerPlaneWithMaxCapacity()  {
         Airport airport = new Airport(planes);
         PassengerPlane expectedPlaneWithMaxPassengersCapacity = airport.getPassengerPlaneWithMaxPassengersCapacity();
-        Assert.assertTrue(expectedPlaneWithMaxPassengersCapacity.equals(planeWithMaxPassengerCapacity));
+        Assert.assertEquals(expectedPlaneWithMaxPassengersCapacity, planeWithMaxPassengerCapacity);
     }
 
+
     @Test
-    public void testNextPlaneMaxLoadCapacityIsHigherThanCurrent() throws NonMilitaryPlainException {
+    public void testHasAtLeastOneBomberInMilitaryPlanes(){
         Airport airport = new Airport(planes);
-        airport.sortByMaxLoadCapacity(planes);
-        List<? extends Plane> planesSortedByMaxLoadCapacity = airport.getPlanes();
-        boolean nextPlaneMaxLoadCapacityIsHigherThanCurrent = true;
-
-        {
-            if (!(testComparasionNextPlaneWithCurrent(planesSortedByMaxLoadCapacity, nextPlaneMaxLoadCapacityIsHigherThanCurrent)))
-                Assert.fail("Test failed!");
+        List<MilitaryPlane> bomberMilitaryPlanes = airport.getMilitaryPlanesByType(MilitaryType.BOMBER);
+        Assert.assertTrue(isBomberMilitaryPlane(bomberMilitaryPlanes));
 
-        }
-        Assert.assertTrue(testComparasionNextPlaneWithCurrent(planesSortedByMaxLoadCapacity, nextPlaneMaxLoadCapacityIsHigherThanCurrent));
     }
 
 
     @Test
-    public void testHasAtLeastOneBomberInMilitaryPlanes() throws NonMilitaryPlainException {
+    public void testExperimentalPlanesHasClassificationLevelHigherThanUnclassified()  {
         Airport airport = new Airport(planes);
-        List<MilitaryPlane> bomberMilitaryPlanes = airport.getMilitaryPlanesByType(MilitaryType.BOMBER);
-        boolean isbomberMilitaryPlane = false;
-        if (!(testisBomberMilitaryPlane(bomberMilitaryPlanes, isbomberMilitaryPlane))) {
-            Assert.fail("Test failed!");
-        }
-        Assert.assertTrue(testisBomberMilitaryPlane(bomberMilitaryPlanes, isbomberMilitaryPlane));
+        List<ExperimentalPlane> experimentalPlanes = airport.getExperimentalPlanes();
+        Assert.assertTrue(!(isPlaneUnclassified(experimentalPlanes)));
     }
 
 
     @Test
-    public void testExperimentalPlanesHasClassificationLevelHigherThanUnclassified() throws NonMilitaryPlainException {
+    public void testNextPlaneMaxLoadCapacityIsHigherThanCurrent() {
         Airport airport = new Airport(planes);
-        List<ExperimentalPlane> experimentalPlanes = airport.getExperimentalPlanes();
-        boolean hasUnclassifiedPlanes = false;
-        Assert.assertFalse(testisPlaneUnclassified(experimentalPlanes, hasUnclassifiedPlanes));
+        airport.sortByMaxLoadCapacity(planes);
+        List<? extends Plane> planesSortedByMaxLoadCapacity = airport.getPlanes();
+        Assert.assertTrue(comparasionNextPlaneWithCurrent(planesSortedByMaxLoadCapacity));
+    }
+
+    @Test
+    public void testGetTransportMilitaryPlanes() {
```

Comment:
naming!
test tests method in which way?
since we don't test any specific feature, just a method return, it could be written as methodName_ExpectedBehavior
ex: getMilitaryPlanesByType_returnsNotEmptyList

FYI: https://dzone.com/articles/7-popular-unit-test-naming